README.md

```diff
@@ -29,13 +29,24 @@ yarn add @vaxxnz/nzcp
 
 ## Usage
 
+### Online
+
 ```javascript
 import { verifyPassURI } from "@vaxxnz/nzcp";
 
-// Verify a New Zealand COVID-19 Pass
+// Verify a live New Zealand COVID-19 Pass, resolving the DID document
 const result = await verifyPassURI("NZCP:/1/2KCEVIQEIVVWK6...");
 ```
 
+### Offline
+
+```javascript
+import { verifyPassURI } from "@vaxxnz/nzcp";
+
+// Verify a live New Zealand COVID-19 Pass, using a prefetched DID document
+const result = verifyPassUriOffline("NZCP:/1/2KCEVIQEIVVWK6...");
```

Comment:
Typo with case, this should be `verifyPassURIOffline`.

Reply:
Nice catch, thanks @madleech